components/AvalancheForecastZoneMap.tsx

```diff
@@ -415,6 +441,8 @@ const AvalancheForecastZoneCards: React.FunctionComponent<{
           transform: [controller.getTransform()],
         },
       ]}
+      onScroll={handleScroll}
+      scrollEventThrottle={200}
```

Comment:
might be worth a quick comment why you need this - is it to prevent the highlight from zooming around the map when you scroll? 

also you might want to look at https://lodash.com/docs/4.17.15#debounce for changing the selection only when panning has settled

Reply:
It just didn't seem necessary to fire this 1k times a second, is all. I actually realized that we have a bigger problem - `onScroll` doesn't distinguish between a user panning the thing and us programatically calling `scrollToIndex()`, so sometimes but not always when you push a zone and we scroll through many others on the way there, this `onScroll` would interrupt that and everything would get weird. Using `onMomentumScrollEnd` makes it look really laggy - it waits until scrolling is entirely finished before firing. Not sure how to proceed.